command/genesis/params.go

```diff
@@ -142,12 +144,15 @@ func (p *genesisParams) validateFlags() error {
 		return errInvalidEpochSize
 	}
 
-	// Validate min and max validators number
-	if err := command.ValidateMinMaxValidatorsNumber(p.minNumValidators, p.maxNumValidators); err != nil {
-		return err
+	// Validate validatorsPath only if validators information were not provided via CLI flag
+	if len(p.validators) == 0 {
+		if _, err := os.Stat(p.validatorsPath); errors.Is(err, os.ErrNotExist) {
```

Comment:
Shouldn't we handle all errors?

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1389/commits/b9d96c8adb9c3a856fc88b242dce5f3d0bdcdcad